pkg/network/encoding/usm_windows_test.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+// +build windows
+
+package encoding
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/network"
+	"github.com/DataDog/datadog-agent/pkg/network/types"
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUSMLookup(t *testing.T) {
```

Comment:
sorry to be picky, but let's add a NAT use case to cover all options.
I do suggest to use the matrix testing paradigm, to ease adding more use-cases.

[example](https://github.com/DataDog/datadog-agent/blob/93e5e1a506dff6f875b49a25c805b851a77b716a/pkg/tagger/global_test.go#L18C1-L60)

Reply:
I don't think that behavior is specified for Windows as we don't use the `IPTranslation` field when generating connection keys:
https://github.com/DataDog/datadog-agent/blob/pedro%2Fencoding-improvements/pkg/network/event_common_windows.go#L21-L23

Reply:
so let's add a dedicated tests only for the linux version to cover it
wdyt?

Reply:
The linux version does have a NAT test-case already:
https://github.com/DataDog/datadog-agent/blob/091854b3e294742f234f00678099005ddc73179e/pkg/network/encoding/usm_lookup_test.go#L54-L94
Did you have something else in your mind?